hi_hello.py

```diff
@@ -0,0 +1,16 @@
+def words_length(sentence) -> iter:
```

Comment:
Use type hints
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/